src/backend/utils/misc/faultinjector.c

```diff
@@ -2,7 +2,7 @@
  *
  * faultinjector.c
  *	  GP Fault Injectors are used for Greenplum internal testing only.
- * 
+ *
```

Comment:
I think it's worth removing formatting changes here and below that aren't relevant to this patch.